.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - uses: krdlab/setup-haxe@v1
```

Comment:
Do you really need haxe available when running `configlet`?

Reply:
I am fine with removing the setup-haxe.  I'll add it back if I get some test generation setup.